forc/src/ops/forc_fmt.rs

```diff
@@ -222,8 +222,8 @@ name = "Fuel example project"
 
 
 [dependencies]
-core = {git="http://github.com/FuelLabs/sway-lib-core"}
```

Comment:
you should add more identation so we actuallly check that the version number gets indented correctly

Reply:
This line verifies that missing whitespace will be added if needed. See the line below this for removing extra whitespace.

Reply:
oh ok, smarter than what I had done!